issue_12/script.js

```diff
@@ -0,0 +1,75 @@
+function startLoading() {
+    const loadingImg = document.createElement('img');
+    loadingImg.src = 'loading-circle.gif';
+    loadingImg.alt = 'ローディング画像';
+    document.getElementById('js-loading').appendChild(loadingImg);
+}
+
+function removeLoading() {
+    document.getElementById('js-loading').remove();
+}
+
+function createLists(resolvedData) {
+    const ul = document.getElementById('js-ul');
+    const fragment = document.createDocumentFragment();
+    for (const item of resolvedData) {
+        const li = document.createElement('li');
+        const a = document.createElement('a');
+        const img = document.createElement('img');
+
+        a.href = item.to;
+        a.textContent = item.text;
+        img.src = item.img;
+        img.alt = item.alt;
+
+        fragment.appendChild(li).appendChild(a).appendChild(img);
+    }
+    ul.appendChild(fragment);
+}
+
+function displayMessage(error) {
+    const ul = document.getElementById('js-ul');
+    ul.textContent = error;
+}
```

Comment:
IMO
I thought that defining a variable for something that is only used once is a waste of memory, so there is no need to define it, what do you think?
```suggestion
function displayMessage(error) {
  document.getElementById("js-ul").textContent = error;
}
```

Reply:
It certainly doesn't seem very necessary to define a variable when it is only used once. Fixed.
Also, I am learning about variable and argument names. 
Thanks for sharing :raised_hands:
https://github.com/yoshikawa-akr/js-lessons/pull/12/commits/b2255659d8709af27a975dbce5db94e654029c9b